library/std/src/process.rs

```diff
@@ -606,6 +606,22 @@ impl Command {
     /// but this has some implementation limitations on Windows
     /// (see issue #37519).
     ///
+    /// [`Command::new`] is only intended to accept the path of the program. If you pass a program
+    /// path along with arguments such as `ls -l` for the program `ls` and argument `-l`, it will
+    /// try to search for `ls -l` literally.
+    ///
+    /// ```no_run (example demonstrating incorrect usage)
+    /// use std::process::Command;
+    ///
+    /// // Does not launch `ls`, will try to launch a program named `ls -l` literally.
+    /// Command::new("ls -l")
```

Comment:
this is the first code example in the documentation for the method, that seems a bit unfortunate, I can imagine someone getting a bit confused at first

Reply:
To avoid this confusion (and get it out of the "main" focus of the doc comment), I moved it into a `# Caveats` section.